zuul.d/projects.yaml

```diff
@@ -2,108 +2,5 @@
-      - cifmw-molecule-ci_gen_kustomize_values
-      - cifmw-molecule-ci_local_storage
-      - cifmw-molecule-ci_lvms_storage
-      - cifmw-molecule-ci_metallb
+      - podified-multinode-edpm-deployment-crc-bootstrap-staging-ironic
       - cifmw-molecule-ci_multus
-      - cifmw-molecule-ci_netconfig
-      - cifmw-molecule-ci_network
-      - cifmw-molecule-ci_nmstate
-      - cifmw-molecule-ci_setup
-      - cifmw-molecule-cifmw_block_device
-      - cifmw-molecule-cifmw_ceph_client
-      - cifmw-molecule-cifmw_ceph_spec
-      - cifmw-molecule-cifmw_cephadm
-      - cifmw-molecule-cifmw_create_admin
-      - cifmw-molecule-cifmw_external_dns
-      - cifmw-molecule-cifmw_ntp
-      - cifmw-molecule-cifmw_test_role
-      - cifmw-molecule-compliance
-      - cifmw-molecule-config_drive
-      - cifmw-molecule-copy_container
-      - cifmw-molecule-deploy_bmh
-      - cifmw-molecule-devscripts
-      - cifmw-molecule-discover_latest_image
-      - cifmw-molecule-dlrn_promote
-      - cifmw-molecule-dlrn_report
-      - cifmw-molecule-dnsmasq
-      - cifmw-molecule-edpm_build_images
-      - cifmw-molecule-edpm_deploy
-      - cifmw-molecule-edpm_deploy_baremetal
-      - cifmw-molecule-edpm_kustomize
-      - cifmw-molecule-edpm_prepare
-      - cifmw-molecule-env_op_images
-      - cifmw-molecule-hci_prepare
-      - cifmw-molecule-hive
-      - cifmw-molecule-idrac_configuration
-      - cifmw-molecule-install_ca
-      - cifmw-molecule-install_openstack_ca
-      - cifmw-molecule-install_yamls
-      - cifmw-molecule-kustomize_deploy
-      - cifmw-molecule-libvirt_manager
-      - cifmw-molecule-manage_secrets
-      - cifmw-molecule-mirror_registry
-      - cifmw-molecule-nat64_appliance
-      - cifmw-molecule-networking_mapper
-      - cifmw-molecule-openshift_adm
-      - cifmw-molecule-openshift_login
-      - cifmw-molecule-openshift_obs
-      - cifmw-molecule-openshift_provisioner_node
-      - cifmw-molecule-openshift_setup
-      - cifmw-molecule-operator_build
-      - cifmw-molecule-operator_deploy
-      - cifmw-molecule-os_must_gather
-      - cifmw-molecule-os_net_setup
-      - cifmw-molecule-ovirt
-      - cifmw-molecule-pkg_build
-      - cifmw-molecule-podman
-      - cifmw-molecule-polarion
-      - cifmw-molecule-registry_deploy
-      - cifmw-molecule-repo_setup
-      - cifmw-molecule-reportportal
-      - cifmw-molecule-reproducer
-      - cifmw-molecule-rhol_crc
-      - cifmw-molecule-run_hook
-      - cifmw-molecule-set_openstack_containers
-      - cifmw-molecule-shiftstack
-      - cifmw-molecule-ssh_jumper
-      - cifmw-molecule-sushy_emulator
-      - cifmw-molecule-switch_config
-      - cifmw-molecule-tempest
-      - cifmw-molecule-test_deps
-      - cifmw-molecule-test_operator
-      - cifmw-molecule-tofu
-      - cifmw-molecule-update
-      - cifmw-molecule-update_containers
-      - cifmw-molecule-validations
-      - cifmw-molecule-virtualbmc
-    github-experimental-trigger:
```

Comment:
are we disabling all the check jobs and the downstream trigger jobs?

Reply:
@frenzyfriday I guess this is a leftover from @lewisdenny testing.
@lewisdenny, please, remove this file from the PR.

Reply:
Sorry about that one guys, checked out the file from main now.